src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -159,32 +159,46 @@ export const View = (props: Props) => {
 
   const exposureCardElems = filteredExposures.map(
     (exposure: ScanResult | HibpLikeDbBreach, index) => {
+      let email;
+      // Get the email assosciated with breach
+      if (!isScanResult(exposure)) {
+        props.userBreaches.breachesData.verifiedEmails.forEach(
```

Comment:
Hm, I bet this could be a little more concise with `.filter` but it looks correct :)